x-pack/plugins/maps/public/classes/sources/es_geo_line_source/layer_wizard.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { CreateSourceEditor } from './create_source_editor';
+import { ESGeoLineSource, geoLineTitle } from './es_geo_line_source';
+import { LayerWizard, RenderWizardArguments } from '../../layers/layer_wizard_registry';
+import { ColorDynamicOptions } from '../../../../common/descriptor_types';
+import {
+  COLOR_MAP_TYPE,
+  FIELD_ORIGIN,
+  LAYER_WIZARD_CATEGORY,
+  STYLE_TYPE,
+  VECTOR_STYLES,
+} from '../../../../common/constants';
+import { getDefaultDynamicProperties } from '../../styles/vector/vector_style_defaults';
+import { VectorStyle } from '../../styles/vector/vector_style';
+import { VectorLayer } from '../../layers/vector_layer/vector_layer';
+import { getIsGoldPlus } from '../../../licensed_features';
+import { TracksLayerIcon } from '../../layers/icons/tracks_layer_icon';
+
+export const geoLineLayerWizardConfig: LayerWizard = {
+  categories: [LAYER_WIZARD_CATEGORY.ELASTICSEARCH],
+  description: i18n.translate('xpack.maps.source.esGeoLineDescription', {
+    defaultMessage: 'Connect points into lines',
+  }),
+  disabledReason: i18n.translate('xpack.maps.source.esGeoLineDisabledReason', {
+    defaultMessage: '{title} requires a Gold license.',
+    values: { title: geoLineTitle },
+  }),
+  icon: TracksLayerIcon,
+  getIsDisabled: () => {
+    return !getIsGoldPlus();
+  },
+  renderWizard: ({ previewLayers }: RenderWizardArguments) => {
+    const onSourceConfigChange = (
+      sourceConfig: {
+        indexPatternId: string;
+        geoField: string;
+        splitField: string;
+        sortField: string;
+      } | null
+    ) => {
+      if (!sourceConfig) {
+        previewLayers([]);
+        return;
+      }
+
+      const defaultDynamicProperties = getDefaultDynamicProperties();
+      const layerDescriptor = VectorLayer.createDescriptor({
```

Comment:
this is minor, but consider setting `alpha: 1`.

Real world track data would probably follow roads. Because of the default color-palette, and the small line-width, tracks don't really show up.

e.g. default result of NYC-bustracks on the EMS-baselayer.

![image](https://user-images.githubusercontent.com/1833023/100951222-23065880-34dc-11eb-9abf-0aeeb947b986.png)